Generators/src/main/java/io/deephaven/pythonPreambles/plotV2.py

```diff
@@ -78,6 +78,10 @@ def _convert_j(name: str, obj: Any, types: List) -> Any:
         else:
             return obj.value
     elif isinstance(obj, Sequence):
+        if not obj or not isinstance(obj[0], (bool, int, float)):
+            return obj
```

Comment:
I hate checking just the first item of a sequence and calling it "good". I also don't really understand what's going on here... in some cases we are return `obj` or unwrapping it... but why are we doing something with numpy?